### ISSUE TITLE
Ice connection state change delegate method is not triggering

### ISSUE THREAD

**Issue by sudhamoorthi** [Ankit-Aggarwal/SwiftyWebRTC#30]
The following method is not at all triggered for any of the state like below one. 
 RTCIceConnectionStateNew,

  RTCIceConnectionStateChecking,

  RTCIceConnectionStateConnected,

  RTCIceConnectionStateCompleted,

  RTCIceConnectionStateFailed,

  RTCIceConnectionStateDisconnected,

  RTCIceConnectionStateClosed,

  RTCIceConnectionStateCount,

  public func peerConnection(_ peerConnection: RTCPeerConnection, didChange newState: RTCIceConnectionState) {

        self.delegate?.rtcClient(client: self, didChangeConnectionState: newState)

    }

 

But at the same time the below delegate is triggering and i am getting ice candidates. 

 public func peerConnection(_ peerConnection: RTCPeerConnection, didGenerate candidate: RTCIceCandidate) {

        self.delegate?.webrtcClient(client: self, didGenerateIceCandidate: candidate)

    }

The peer connection is not established in this case. Kindly do the needful on this.

**Comment by sudhamoorthi** [comment]
Kindly do the needful. i am not able to proceed further

**Comment by ShaikBaji1506** [comment]
@sudhamoorthi  are you able to make connections in different networks from device to device ?

**Comment by sudhamoorthi** [comment]
yes. connection is established between different device in different networks.

**Comment by ShaikBaji1506** [comment]
@sudhamoorthi  without triggering  RTCIceConnectionStateChecking,

RTCIceConnectionStateConnected,

RTCIceConnectionStateCompleted,

RTCIceConnectionStateFailed,

RTCIceConnectionStateDisconnected,

RTCIceConnectionStateClosed,

RTCIceConnectionStateCount,

the above method
 how are you able to connect (make a connection video call) can you please ping me to mail : shaikbaji1911@gmail.com I have some query
